rest-client/src/test/java/org/sagebionetworks/bridge/rest/ErrorResponseInterceptorTest.java

```diff
@@ -122,24 +125,25 @@ public void badRequest400() throws Exception {
     @Test
     public void unrecognizedUrl400() throws Exception {
         // Play sends responses like these if you call an unrecognized URL.
-        doReturn(400).when(response).code();
-        doReturn("<html><body>Play returns HTML instead of JSON</body></html>").when(body).string();
-        doReturn("Play Framework does not recognize this URL").when(response).message();
+        doReturn(404).when(response).code();
+        String json = Tests.unescapeJson("{'message':'Endpoint not found.','type':'EndpointNotFoundException'}");
+        doReturn(json).when(body).string();
```

Comment:
Okay good point, let me use a combination of tactics that will preserve HTML + status code